src/components/Login/userReducer/userReducer.ts

```diff
@@ -0,0 +1,49 @@
+import { UserActions, UserActionTypes, UserState } from "../types/user";
+
+const initialState: UserState = {
+  isFetching: false,
+  isLoggedIn: false,
+  error: false,
+  account: {
+    avatar: {
+      gravatar: {
+        hash: "no avatar",
+      },
+      tmdb: {
+        avatar_path: "no avatar",
```

Comment:
why "no avatar"? just empty string should be enough

Reply:
change